torchutils/models.py

```diff
@@ -18,10 +22,28 @@ def get_model_param_count(model):
     """
 
     _validate_param(model, 'model', 'model')
+    _validate_param(param_type, 'param_type', 'str')
```

Comment:
If using a string argument, you'll need to also validate that the input string is valid.

```
if param_type not in {'total,'train','nontrain'}:
    raise
```

I think it is better to use  boolean since it is simpler and you don't need the above check. Also user won't have to figure out weather the string is 'train', 'trainable', 'nontrain', 'non-train'. There can be too many possible options. Better to just keep True or False.